tools/extra/opae.io/opae/io/utils.py

```diff
@@ -321,6 +321,8 @@ def register(self, offset, name='csr', bits=[('value', c_uint64, 64)]):
                 (0x8086, 0x09c5) : "Intel PAC A10 GX VF",
                 (0x8086, 0x0b2b) : "Intel PAC D5005",
                 (0x8086, 0x0b2c) : "Intel PAC D5005 VF",
+                (0x8086, 0xaf00) : "Intel PAC D5005",
+                (0x8086, 0xaf01) : "Intel PAC D5005 VF",
```

Comment:
af00 and af01 are not D5005 devices.

Reply:
#define PCIE_DEVICE_ID_INTEL_PAC_D5005		0x0B2B
#define PCIE_DEVICE_ID_INTEL_PAC_D5005_VF	0x0B2C
#define PCIE_DEVCIE_ID_INTEL_AF00		0xaf00

The device ID of D5005 should be 0x0b2b，for VF of D5005 is 0x0b2c

Reply:
If the goal is to add 0xaf00 and 0xaf01, then the associated strings should read N6010 rather than D5005.